pkg/universe.dagger.io/python/test/test.cue

```diff
@@ -10,50 +10,57 @@ import (
 dagger.#Plan & {
 	actions: test: {
 
-		// Run with a custom path to python
-		customPath: {
+		"python.cue": {
+			// Run with a custom path to python
+			customPath: {
 
-			// python:3.10-alpine has both python3 and python in the PATH
-			_image: docker.#Pull & {
-				source: "python:3.10-alpine"
-			}
-			run: python.#Run & {
-				input: _image.output
-				command: name:    "python"
-				script: contents: #"print("Hello, world!")"#
+				// python:3.10-alpine has both python3 and python in the PATH
+				_image: docker.#Pull & {
+					source: "python:3.10-alpine"
+				}
+				run: python.#Run & {
+					input: _image.output
+					command: name:    "python3"
+					script: contents: #"print("Hello, world!")"#
+				}
+				// This needs no output test because it is only testing that the command runs
 			}
-			// This needs no output test because it is only testing that the command runs
-		}
 
-		// Run a script from source directory + filename
-		runFile: {
+			// Run a script from source directory + filename
+			runFile: {
 
-			dir:   _load.output
-			_load: core.#Source & {
-				path: "./data"
-				include: ["*.py"]
+				dir:   _load.output
+				_load: core.#Source & {
+					path: "./data"
+					include: ["*.py"]
```

Comment:
Nit: You could read it with client API instead of `core.#Source`

Reply:
I agree that client API is better, but am hesitant to change this because I don't fully understand the context I'm working in. `core.#Source` is always relative to the test.cue, but `client.filesystem` is relative to the cwd you run `dagger` from.

I tried to hack it together and test with `make universe-test` but it hung forever in a busy loop on I/O and I'm not sure why and ran out of time and patience with this PR :). I'm willing to accept it, since universe/bash also uses core.#Source for its tests. I wouldn't be opposed to taking on a separate task to update a bunch of universe packages to be a little more idiomatic, though.

Reply:
@TomChv `core.#Source` is actually prefered in tests, for the reason @NotTheEconomist said. We should use it more in our tests.

Reply:
Correct

Reply:
Okay! Then we should do it in all universe tests since no one should test the client API